### ISSUE TITLE
Unable to retrieve child resources.

### ISSUE THREAD

**Issue by braxxi** [microsoft/AzureStorageExplorer#3711]
**Storage Explorer Version**: 1.15.1
**Platform/OS**: Win10
**Architecture**: x64
**Regression From**:  didnt work on previous version too
## Steps to Reproduce ##
With Azure Storage Explorer open the blob storage

## Expected Experience ##
be able to open blob storage to view the content

## Actual Experience ##
Cant open blob storage, cant do anything there

## Additional Context ##
Happens quite often and randomly. Have several storage accounts. 3 out of 4 works, one not. Another time 2 works and 2 not. 
Have access through portal but not vis storage explorer.
![Capture123](https://user-images.githubusercontent.com/46719801/97009846-e93d4c00-1544-11eb-9a7d-55037d626745.PNG)

**Comment by ahmaibr** [comment]
https://social.technet.microsoft.com/wiki/contents/articles/53393.azure-storage-explorer-troubleshooting-unable-to-retrieve-child-resources-or-the-request-action-could-not-be-completed.aspx

tried as per above link and its still not working

**Comment by ahmaibr** [comment]
https://social.msdn.microsoft.com/Forums/en-US/41b077a6-3c47-4da4-8ecc-a5a91848ffb5/microsoft-azure-storage-explorer-unable-to-retrieve-child-resources?forum=windowsazuredata

tried as per above link and its still not working

**Comment by MRayermannMSFT** [comment]
@ahmaibr we think this is most likely a storage firewall issue. Do you have a firewall enabled for any of the storage accounts that are causing you issues? Does this error consistently reproduce only from certain machines/networks?

Alternatively this could be an issue with your permissions. Are you using RBAC or do you have permissions to list account keys?

**Comment by MRayermannMSFT** [comment]
@braxxi what we said to @ahmaibr applies to you as well.

**Comment by edtiong** [comment]
Facing the same issue. Didn't have issues before - came up suddenly. 

![image](https://user-images.githubusercontent.com/53787081/98714933-7bbc5880-2357-11eb-8df6-dd881e4e0faa.png)

**Comment by edtiong** [comment]
@MRayermannMSFT  Can you provide me with concrete steps to debug? how to check permission/firewall etc

**Comment by braxxi** [comment]
Finally got this resolved by setting up a proxy.  It was matter of a local settings. No changes in a firewall

**Comment by MRayermannMSFT** [comment]
@edtiong for your error:

- if you have any accounts that need reauthentication, please reauthenticate them
- if the error persists, try disabling disks by going to Settings -> Services -> Disks, and then un checking the check box.

**Comment by MRayermannMSFT** [comment]
I known editong has become unblocked as they commented on another issue saying they were good to go. Everyone else on this issue is either unblocked or has not recently replied. Closing.
